### PR TITLE
Added getCallback check

### PR DESCRIPTION
**Pull request by danmorton**
Avoid possible undefined target error.

### CODE REVIEW

**Comment by jnbn** [comment]
+1

**Comment by UWouldlLose** [comment]
Just ran across this as well.

**Comment by ain** [comment]
:+1: for a merge.

**Comment by albell** [comment]
Can this be merged soon and pushed to the CDN? This is throwing up a lot of errors for me. Could we possibly get some idea of the timeline on this? Thank you.

**Comment by ain** [comment]
The repo has had no update in half a year and is actually pretty badly organised. I'm amazed by Vimeo's QA on this one, to be honest.

**Comment by ain** [comment]
It's now 1 month since my last comment and it's a blocker-grade issue. Hello, guys at Vimeo, any update possible?

**Comment by githubuser5859** [comment]
Not merged yet? I found the problem before coming here. Used a try..block but this will work, as well. When will it be merged and pushed to the CDN?

File: javascript/froogaloop.js [hunk 1]
```diff
@@ -208,7 +208,7 @@ var Froogaloop = (function(){
      * Retrieves stored callbacks.
      */
     function getCallback(eventName, target_id) {
-        if (target_id) {
+        if (target_id && eventCallbacks[target_id] != undefined) {
```
Comment (nikolas) [inline javascript/froogaloop.js]: I would change this to: `if (target_id && typeof eventCallbacks[target_id] !== 'undefined') {`

**Comment by SamPotts** [comment]
+1. I was just about to do a pull request for this myself then saw this :+1:

**Comment by bdougherty** [comment]
This is a duplicate of #37

**Pull request closed without merging**